BIDSTools/CreatBIDSDataset.py

```diff
@@ -5,7 +5,7 @@
 import json
 from elab_bridge import server_interface
 from BidsEmptyRepositoryGenerator import Generator
-from writing_agnosticfile import *
+from Writing_modality_Agnostics_files import *
```

Comment:
thanks for the change! if you want to be perfect, you should try to respect some conventions for the naming of the modules like these... maybe WritingModalityAgnosticFiles (with no underscores, and capital letters at each word)? To be checked...